com.unity.shadergraph/Editor/Serialization/MultiJsonInternal.cs

```diff
@@ -361,7 +361,9 @@ public static JsonObject CreateInstance(string typeString)
             {
                 return new UnknownJsonObject(typeString);
             }
-            return (JsonObject)Activator.CreateInstance(type, true);
+            var output = (JsonObject)Activator.CreateInstance(type, true);
+            output.ChangeVersion(0);
```

Comment:
Why does it immediately change version to 0 ?   Comment needed if it's meant to do that.

Reply:
Yes, right, this deals with the fact that this instance creation is meant to be a "blank" instance used in deserialization. The main place this is a problem is upgrading, since version number didnt exist before and so deserializing a value into an instace will not overwrite the version and leave it as latest. By explicitly setting it to zero, if a value is deserialized and brought in and has no version, then it will correctly be treated as version 0